source/importers/SynchronizeColorsTokens.ts

```diff
@@ -0,0 +1,218 @@
+const fsT = require('fs');
+const axios = require("axios");
+const tokensFilesConfig = require('./configs/SynchronizeColorsTokensConfig.json');
+
+interface IResultFileJsonObject {
+	id: string;
+	label: string;
+	value: string;
+}
+
+interface IResultJsonObject  extends IResultFileJsonObject {
+	weight: number;
+}
+
+interface configFileObject {
+	destination: string,
+	destinationJson: string
+}
+
+interface ITokenJsonObject {
+	id: string;
+	label: string;
+	value: string;
+}
+
+const ImportColorsRaw = (
+	binValues: configFileObject,
+	jsonColors: Array<any>
+) => {
+	let hexToCssVariable: Object = {};
+	try {
+		let result: Array<string> = [];
+		let temporaryColorsJson = {};
+		let resultColorsJson = {};
+
+		result.push(':root {');
+
+		jsonColors.forEach(obj => {
+			const patternColorsToProcess = /RAW\/|theme/i;
+			const patternColorsToIgnore = /deprecated|Pattern/i;
+			const patternTheme = /theme/i;
+			if (obj.name.match(patternColorsToProcess) && obj.name.match(patternColorsToIgnore) === null) {
+				const nameSplit = obj.name.split('/');
+				let colorName = (nameSplit[2] === undefined) ? nameSplit[1] : nameSplit[2];
+				console.log(colorName);
+				obj.values.hex = obj.values.hex.replace(/FFFFFF/gi, function(){
```

Comment:
we already found a way to handle all colors, can we apply it here?

Reply:
Updated.